src/custom/pages/Faq/index.tsx

```diff
@@ -640,7 +644,7 @@ export default function Faq() {
           </h3>
 
           <p>
-            Signing a message incurs no gas cost and is therefore free to the user. When placing an order, the protocol
+            Signing a message incurs no gas cost and is, therefore free to the user. When placing an order, the protocol
```

Comment:
I find this comma strange. Maybe ask a native?

Reply:
Reads more natural without the comma.